### ISSUE TITLE
Never got 2FA code

### ISSUE THREAD

**Issue by zEw0** [SideStore/SideStore#260]
### Describe the bug

When I try to refresh or update an app it asks for a 2FA code which never shows up.

### Instructions to reproduce

1. Refresh or update an app

### What version of SideStore are you using?

Sidestore 0.1.1

### Other info

I am currently using ios 16.2 and also a custom anisette server. Oh and im also using adp (advanced data protection). One more thing, I was able to use adp with 2fa on sidestore once but once I completely removed and reinstalled sidestore (cause I forgot to refresh sidestore) and updated the custom anisette server to the latest version, i never got adp and 2fa to work.

**Comment by Spidy123222** [comment]
This isn't a Sidestore problem. Make a GitHub issue on the anisette server programs GitHub. ADP isn't supported on most anisette servers/programs.

**Comment by naturecodevoid** [comment]
Duplicate of #232

You are using a very outdated version of SideStore; anisette servers have also needed some updates recently to work with ADP. Anisette fixes are currently in the works, but they aren't quite ready yet, since there are some things that need to be done before being added to nightly.

**Comment by naturecodevoid** [comment]
> This isn't a Sidestore problem. Make a GitHub issue on the anisette server programs GitHub. ADP isn't supported on most anisette servers/programs. 

Wow, you responded just before me, lol.

**Comment by Spidy123222** [comment]
> > This isn't a Sidestore problem. Make a GitHub issue on the anisette server programs GitHub. ADP isn't supported on most anisette servers/programs. 
> 
> Wow, you responded just before me, lol.

Discord GitHub bot speedy.

**Comment by zEw0** [comment]
> Duplicate of #232
> 
> You are using a pretty outdated version of SideStore; anisette servers have also needed some updates recently to work with ADP. Anisette fixes are currently in the works, but they aren't quite ready yet, since there are some things that need to be done before being added to nightly.
> 
> As for not receiving 2FA code, I don't think it's an issue on our side.

Oh really? I didn't know lol cause I swear I literally just downloaded that sidestore ipa from the official website yesterday.

**Comment by naturecodevoid** [comment]
0.1.1 is a bit outdated, we are currently on 0.3.0

**Comment by Spidy123222** [comment]
> 0.1.1 is a bit outdated, we are currently on 0.3.0

0.1.1 is current version and nightly is on 0.3.0 atm. The beta builds will turn to 0.3.0 some point once more things ironed out like what was discussed in discord.
